src/applications/gi-sandbox/reducers/search.js

```diff
@@ -19,6 +23,11 @@ const INITIAL_STATE = {
   geocode: null,
   geocodeInProgress: false,
   geolocationInProgress: false,
+  streetAddress: {
```

Comment:
Is this part of `query` or `location`?
It should be inside the corresponding search tab's object or in the query object.